client/src/components/Display/MoneyDisplay/BottomContent/BottomContent.module.less

```diff
@@ -0,0 +1,12 @@
+.bottomSection {
+  width: 100%;
+  padding-top: 40px;
+  align-items: center;
```

Comment:
Does this `align-items: center` do anything when we don't have `display: flex` on this element?

Reply:
Yes, because it's on antd element. But i moved it to the element's props